lib/diagnostic.rb

```diff
@@ -108,10 +110,10 @@
 # Use shorthand to make this one line long.
 # Write your response as a Ruby string.
 
-Diagnostic.person_hash = 'your answer here'
+Diagnostic.person_hash = 'person = { favorite_number: 7, first_name: "spencer" }'
 
 # Set a default return value to your `person` hash that concatenates the value
 # of person[:first_name] and a message about them.
 # Write your response as a Ruby string.
 
-Diagnostic.hash_default = 'your answer here'
+Diagnostic.hash_default = Hash.new { |hash, key| hash[key] = "Key #{first_name} is now part of person hash" }
```

Comment:
We can set a default return value on the hash like so
`person.default = "#{person[:first_name]} is great"`